Krivorotova Tatyana/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Cubes</title>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+        <link href="https://fonts.googleapis.com/css?family=Lato:700&display=swap" rel="stylesheet">
+    </head>
+    <body>
+        <div id="widget">
+            <div id="a" class="elem">
+                <img src="images/chicago.png">
+                <span>CHICAGO / 23&deg;</span>
+            </div>
+            <div id="b" class="elem">
+                <img src="images/seattle.png">
+                <span>SEATTLE / 34&deg;</span>
+            </div>
+            <div id="c" class="elem">
+                <img src="images/brooklyn_cloud.png">
+                <span>BROOKLYN / 55&deg;</span>
+            </div>
+            <div id="d" class="elem">
+                <img src="images/portland.png">
+                <span>PORTLAND / 27&deg;</span>
+            </div>
+            <div id="e" class="elem">
```

Comment:
as for me `#e` looks like the main part of the widget and you can change `<div>` to  `<main>`